uber/automated_emails.py

```diff
@@ -97,19 +312,28 @@ def __init__(self, conf):
         AutomatedEmail.__init__(self, Attendee,
                                 subject='{EVENT_NAME} Department Checklist: ' + conf.name,
                                 template='shifts/dept_checklist.txt',
-                                filter=lambda a: a.is_single_dept_head and a.admin_account and days_before(7, conf.deadline) and not conf.completed(a),
+                                filter=lambda a: a.is_single_dept_head and a.admin_account and not conf.completed(a),
+                                when=days_before(7, conf.deadline),
                                 sender=c.STAFF_EMAIL,
                                 extra_data={'conf': conf})
 
-before = lambda dt: bool(dt) and localized_now() < dt
-after = lambda dt: bool(dt) and localized_now() > dt
-days_after = lambda days, dt: bool(dt) and (localized_now() > dt + timedelta(days=days))
 
+"""
+IMPORTANT NOTES:
+
+'ident' is a unique ID for that email category that must not change after emails in that category have started to send.
+
+
+
+#If an 'ident' is not set, 'ident' will default to be the 'subject' line.
+#The reason you don't want this is because we keep track of which emails have been sent by their 'ident', and if the
+
+
+TODO: FIXME: Going forward, every single email category below should have an explicit 'ident' set. However,
```

Comment:
you know what, that makes a lot of sense to just stop formatting it.  mostly trying to avoid changes to c.EVENT_NAME resulting in changes to 'ident's.  I'll change it to be as you suggest.

And yea, eventually over the years we should just keep dropping {EVENT_NAME} from any idents